source/PreferencesPanel.cpp

```diff
@@ -448,6 +448,7 @@ void PreferencesPanel::DrawSettings()
 		"Rehire extra crew when lost",
 		SCROLL_SPEED,
 		"Show escort systems on map",
+		"System map sends move orders",
```

Comment:
I don't really see a need to hide this feature behind a preference. Right-clicking on the orbits panel is a very explicit action which I wouldn't expect to happen by accident. (But I would expect support questions from players that have the preference disabled and wonder why this feature doesn't work.)

Reply:
The default game behavior is that clicking on "radar"-like displays doesn't do anything. Instead of a new preference, we could infer from that preference whether this functionality is enabled.

Reply:
Okay, I just re-read the earlier discussions from 2017. Let's hide this behind a preference indeed.